book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] It would be better if we set both getter and setter for our rentals since we will be writing and reading it